test/run_test.go

```diff
@@ -659,9 +659,9 @@ func TestDisallowedOptionsInConfig(t *testing.T) {
 
 		testshared.NewRunnerBuilder(t).
 			WithConfig(c.cfg).
-			WithArgs(getCommonRunArgs()...).
+			WithArgs("--skip-dirs", "testdata_etc/,pkg/golinters/goanalysis/(checker|passes)").
```

Comment:
I kinda like that this was a shared method before, duplicating like this risks adding maintenance if changed. Maybe move the actual valua to a constant add keep the `--skip-dirs` for readability.

```go
const skipDirsCommand = "testdata_etc/,pkg/golinters/goanalysis/(checker|passes)"

// ...
    WithArgs("--skip-dirs", skipDirsCommand")
```

And while at it maybe add a comment about why we skip those dirs because it's not clear by just reading the line to me.